pysat/instruments/methods/madrigal.py

```diff
@@ -88,9 +88,11 @@ def load(fnames, tag=None, sat_id=None, xarray_coords=[]):
     file_meta = filed['Metadata']['Data Parameters']
     # load up what is offered into pysat.Meta
     meta = pysat.Meta()
-    meta.info = {'acknowledgements': "See 'meta.Experiment_Notes' for " +
-                 "instrument specific acknowledgements\n" + cedar_rules(),
-                 'references': "See 'meta.Experiment_Notes' for references"}
+    meta.acknowledgements = "".join(("See 'meta.Experiment_Notes' for ",
+                                     "instrument specific acknowledgements\n",
+                                     cedar_rules()))
+    meta.references = "See 'meta.Experiment_Notes' for references"
+
```

Comment:
Do these attributes still exist in meta?

Reply:
They don't. Thanks! Carried over from develop fix. I also saw other Madrigal instruments handled things in init while checking out other pulls.

Reply:
I thought I was over at pysatMadrigal. I am getting it sorted out though.